.drone.yml

```diff
@@ -46,7 +46,7 @@ steps:
 
   - name: unit-tests
     # see: docker/dockerfiles/Dockerfile
-    image: codedotorg/code-dot-org:1.8.1
+    image: codedotorg/code-dot-org:add-git-lfs
```

Comment:
Will edit these to be (probably) `1.8.2` and push to dockerhub `codedotorg/code-dot-org:1.8.2` once we're ready for merge.

Reply:
make that 1.9.2

Reply:
make that 1.9.3 😅